control-plane-operator/main.go

```diff
@@ -154,7 +155,13 @@ func NewStartCommand() *cobra.Command {
 			setupLog.Error(err, "unable to detect cluster capabilities")
 			os.Exit(1)
 		}
-
+		lookupContext, lookupContextCancel := context.WithTimeout(ctx, 60*time.Second)
+		activeImage, err := util.LookupActiveContainerImage(lookupContext, kubeClient.CoreV1().Pods(namespace), os.Getenv("POD_NAME"), "control-plane-operator")
```

Comment:
@sjenning @cewong @enxebre @alvaroaleman this will break native process workflows, which I no longer use, but you might. An optional flag (so that we look up if possible and use the flag otherwise) could resolve it, but even in the latter case you would now have to pass a new flag w/ the image...